### ISSUE TITLE
Feature Request: Dump Database

### ISSUE THREAD

**Issue by 7flash** [chiselstrike/chiselstrike#1622]
I would like to be able easily save current database and then restore it on another machine, - what are the options to do it?

**Comment by 7flash** [comment]
Just copied ".chiseld.db" file and restarted server - that works.

![image](https://user-images.githubusercontent.com/4569866/183276764-99106d13-5599-410e-98b5-eea031d1d5b7.png)

**Comment by penberg** [comment]
@7flash Yes, if you are running with SQLite, you can just copy the files.
